modules/ephemeris/src/test/scala/gem/horizons/HorizonsUpdaterApp.scala

```diff
@@ -1,97 +0,0 @@
-// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
-// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
-
-package gem.horizons
-
-import gem.{ EphemerisKey, Log }
-import gem.dao.UserDao
-import gem.enum.Site
-
-import cats.effect.IO
-import cats.implicits._
-
-import doobie._, doobie.implicits._
-
-
-/** A simple test application that drives ephemeris updates.
-  *
-  * {{{
-  *   > ephemeris/test:run gem.horizons.HorizonsUpdater site key ...
-  * }}}
-  *
-  * e.g.,
-  *
-  * {{{
-  *   ephemeris/test:run gem.horizons.HorizonsUpdater GS MajorBody_605 MajorBody_606
-  * }}}
-  */
-object HorizonsUpdaterApp {
```

Comment:
This was a way to drive horizons updating from the sbt command line, but now we have a telnet command so it isn't needed.